src/course-updates/update-form/UpdateForm.test.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import {
+  render,
+  fireEvent,
+  waitFor,
+  act,
+} from '@testing-library/react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import moment from 'moment/moment';
+
+import UpdateForm from './UpdateForm';
+import messages from './messages';
+import { REQUEST_TYPES } from '../constants';
+import { courseHandoutsMock, courseUpdatesMock } from '../__mocks__';
+
+const closeMock = jest.fn();
+const onSubmitMock = jest.fn();
+const addNewUpdateMock = { id: 0, date: moment().utc().toDate(), content: 'Some content' };
+const formattedDateMock = '07/11/2023';
+const contentMock = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
+
+// Mock the tinymce lib
```

Comment:
I think this comment is pointing the obvious. I suggest removing it.

Reply:
Done